packages/yew-functional/src/lib.rs

```diff
@@ -75,30 +73,25 @@ pub trait FunctionProvider {
 pub struct FunctionComponent<T: FunctionProvider + 'static> {
     _never: std::marker::PhantomData<T>,
     props: T::TProps,
-    hook_state: RefCell<Option<HookState>>,
+    hook_state: RefCell<HookState>,
     link: ComponentLink<Self>,
     message_queue: MsgQueue,
 }
 
 impl<T> FunctionComponent<T>
-where
-    T: FunctionProvider,
+    where
+        T: FunctionProvider,
```

Comment:
I think you should run rustfmt, the style should be the original one which `impl` and `where` is on the same indentation level.

Reply:
As the CI shows, the code is already formatted properly. Are you sure this is from the latest commit?

Reply:
Ah, you just updated it. Didn't saw it.